config.yaml

```diff
@@ -18,3 +18,20 @@ options:
       Refer to https://prometheus.io/docs/alerting/latest/notification_examples/ for more details
       on templates.
     default: ""
+  web_external_url:
+    description: |
+      The URL under which Alertmanager is externally reachable (for example,
+      if Alertmanager is served via a reverse proxy).
+      Must include the scheme (http or https).
+      Used for generating relative and absolute links back to
+      Alertmanager itself. If the URL has a path portion, it will be used to
+      prefix all HTTP endpoints served by Alertmanager.
+      If omitted, relevant URL components will be derived automatically.
+      If provided, this should be a complete URI, including scheme, or a
+      fully qualified subpath starting with `/`.
+      If Alertmanager is being served directly from the root of a fully-qualified
+      host or a bare A record, this may be omitted.
+      This configuration option takes precedence over the URL provided over
+      the "ingress" relation.
+    type: string
+    default: ""
```

Comment:
This shouldn't be included when you're using Traefik ingress - this is assigned automatically when related to Traefik, and can be customised by the (human) operator by using `traefik-route-k8s`.

Reply:
You mean drop the entire config option?
- Seems like [traefik-route](https://charmhub.io/traefik-route-k8s) is intended for ingress-per-unit, while here we use ingress-per-app.
- Seems like traefik-route needs to be deployed together with traefik. The idea behind the config option is to support any ingress (we won't be able to charm all ingresses in a timely manner).

Reply:
Updated description.